argo-cd-3.1.yaml

```diff
@@ -140,3 +142,4 @@ test:
         argocd-repo-server --help
         argocd-server version
         argocd-server --help
+        argocd-commit-server --help
```

Comment:
It'd be nice to convert it all to use tw's help-check but I don't consider that blocking.